adijif/clocks/hmc7044.py

```diff
@@ -16,8 +16,8 @@ class hmc7044(hmc7044_bf):
     # r2_divider_max = 4095
     r2_available = [*range(1, 4095 + 1)]
     # n2_divider_min = 8
-    # n2_divider_max = 65535
-    n2_available = [*range(8, 65535 + 1)]
+    # n2_divider_max = 4095
+    n2_available = [*range(8, 4095 + 1)]
```

Comment:
Can we make this unique per solver? Since CPLEX should work over the extended ranges.